common/culture/innovations/wc_cultural_maa_innovations.txt

```diff
@@ -38,10 +99,847 @@ innovation_taming_the_skies = {
+	icon = @dire
+
+	potential = {
+		this.culture_group = culture_group:troll_group
+	}
+
+	unlock_maa = dire_troll
+
+	flag = global_maa
+}
+
+innovation_dragon = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @flying
+
+	potential = {
+		OR = {
+			this.culture_group = culture_group:dragon_group
+			this = culture:dragonmaw
+		}
+	}
+
+	unlock_maa = dragon
+
+	flag = global_maa
+}
+
+innovation_gladiator = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @gladiator
+
+	potential = {
+		this = culture:shattered_hand
+	}
+
+	unlock_maa = gladiator
+
+	flag = global_maa
+}
+
+innovation_troll_ranged = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @headhunter
+
+	potential = {
+		this.culture_group = culture_group:troll_group
+	}
+
+	unlock_maa = headhunter
+
+	flag = global_maa
+}
+
+innovation_hexer = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @hexer
+
+	potential = {
+		OR = {
+			this.culture_group = culture_group:troll_group
+			this.culture_group = culture_group:harpy_group
+			this = culture:saurok
+		}
+	}
+
+	unlock_maa = hexer
+
+	flag = global_maa
+} 
+
+innovation_marauder = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @marauder
+
+	potential = {
+		this = culture:centaur
+	}
+
+	unlock_maa = marauder
+
+	flag = global_maa
+}
+
+innovation_magi = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @magi
+
+	potential = {
+		this = culture:ogre
+	}
+
+	unlock_maa = magi
+
+	flag = global_maa
+}
+
+innovation_naga_army = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @myrmidon
+
+	potential = {
+		this = culture:naga
+	}
+
+	unlock_maa = myrmidon
+	unlock_maa = siren
+	unlock_maa = couatl
+	unlock_maa = sea_giant
+
+	flag = global_maa
+}
+
+innovation_night_elf_aerial = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @chimaera
+
+	potential = {
+		OR = {
+			this = culture:night_elf
+			this = culture:dryad
+		}
+	}
+
+	unlock_maa = chimaera
+	unlock_maa = faerie_dragon
+
+	flag = global_maa
+}
+
+innovation_proto_drake_rider = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @proto
+
+	potential = {
+		this.culture_group = culture_group:vrykul_group
+	}
+
+	unlock_maa = proto_drake_rider
+
+	flag = global_maa
+}
+
+innovation_savage = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @savage
+
+	potential = {
+		OR = {
+			this = culture:bleeding_hollow
+			this = culture:laughing_skull
+		}
+	}
+
+	unlock_maa = savage
+
+	flag = global_maa
+}
+
+innovation_sentinel = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @sentinel
+
+	potential = {
+		OR = {
+			this = culture:night_elf
+			this = culture:highborne
+		}
+	}
+
+	unlock_maa = sentinel
+
+	flag = global_maa
+}
+
+innovation_snow_hunter = {
```

Comment:
True, but they never had in the suggestions. And we don't have a cost for winter modifier atm...

Reply:
> True, but they never had in the suggestions. And we don't have a cost for winter modifier atm... Can work the modifier in, once we have a cost for it.

Costs added. You can add costs too on the UNIT COST.

Reply:
So we need buffs for normal, harsh and mild winter on them now?
>  You can add costs too on the UNIT COST.

I know that I technically could, but I hesitated because I didn't want to accidentally break something^^